src/io/gff.rs

```diff
@@ -77,10 +127,12 @@ impl<'a, R: io::Read> Iterator for Records<'a, R> {
                     strand: strand,
                     frame: frame,
                     attributes: csv::Reader::from_string(attributes)
-                        .delimiter(b'=')
-                        .record_terminator(csv::RecordTerminator::Any(b';'))
+                        .delimiter(delim)
+                        .record_terminator(csv::RecordTerminator::Any(termi))
                         .has_headers(false)
-                        .decode().collect::<csv::Result<HashMap<String, String>>>().unwrap(),
+                        .decode()
+                        .collect::<csv::Result<HashMap<String, String>>>()
```

Comment:
Did you try removing the type bounds here? I think the compiler should be able to infer them because of the attributes type.